sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -807,6 +819,17 @@ public partial class ImageAnalysisSkill : Azure.Search.Documents.Indexes.Models.
         public static bool operator !=(Azure.Search.Documents.Indexes.Models.ImageDetail left, Azure.Search.Documents.Indexes.Models.ImageDetail right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class IndexerCurrentState
+    {
+        internal IndexerCurrentState() { }
+        public string AllDocsFinalChangeTrackingState { get { throw null; } }
```

Comment:
What does this property do?  What are the valid values?  Should there be an extensible enum with possibilities?  Could we think of a better name?  Ditto for the related properties.

Reply:
Do we use `Docs` as an abbreviation anywhere else?

Reply:
I have substituted instances of `Docs` with `Documents`.

Reply:
I am thinking we work with the service team to arrive at better names and update in the next preview.

Reply:
I'll do that.